src/Stratis.Bitcoin/Connection/PeerBanning.cs

```diff
@@ -83,7 +83,7 @@ public void BanPeer(IPEndPoint endpoint, DateTime banUntil)
     public class PeerBanning : IPeerBanning
     {
         /// <summary>A connection manager of peers.</summary>
-        private readonly IConnectionManager connectionManager;
+        private readonly IReadOnlyNetworkPeerCollection networkPeerCollection;
```

Comment:
Old comment no longer holds

Reply:
Same with this.  This code will be reverted back to the previous version.